kernel/configs/fragments/common/bpf.conf

```diff
@@ -0,0 +1,43 @@
+# for tc filters
```

Comment:
How about adding a 1 line comment explaining what the file is for (using the term Berkeley Packet Filter than just bpf)?